CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+### citus v12.1.7 (Feb 6, 2025) ###
+
+* Avoids re-assigning the global pid for client backends and bg workers when the application_name changes (#7791)
```

Comment:
let's use the changelog entry we used for release 13.0.0

https://github.com/citusdata/citus/blob/main/CHANGELOG.md?plain=1#L28-L29

```suggestion
* Fixes a crash that happens because of unsafe catalog access when re-assigning
  the global pid after `application_name` changes (#7791)
```